io.github.KangLin.RabbitRemoteControl.json

```diff
@@ -0,0 +1,241 @@
+                "-DWITH_CLIENT_SDL=OFF",
+                "-DWITH_CLIENT=OFF",
+                "-DWITH_KRB5=ON"
+            ],
+            "cleanup": [
+                "/bin",
+                "/share"
+            ]
+        },
+        {
+            "name": "libvncserver",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/LibVNC/libvncserver/archive/refs/tags/LibVNCServer-0.9.15.tar.gz",
+                    "sha256": "62352c7795e231dfce044beb96156065a05a05c974e5de9e023d688d8ff675d7"
+                }
+            ],
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DWITH_OPENSSL=ON",
+                "-DWITH_GCRYPT=OFF",
+                "-DWITH_EXAMPLES=OFF",
+                "-DWITH_TESTS=OFF"
+            ],
+            "cleanup": [
+                "/bin/*",
+                "/lib/libvncserver.so*"
+            ]
+        },
+        {
+            "name": "cmark",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/commonmark/cmark/archive/refs/tags/0.31.1.tar.gz",
+                    "sha256": "3da93db5469c30588cfeb283d9d62edfc6ded9eb0edc10a4f5bbfb7d722ea802"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DBUILD_TESTING=OFF"
+            ],
+            "cleanup": [
+                "/bin/*"
+            ]
+        },
+        {
+            "name": "RabbitCommon",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/KangLin/RabbitCommon.git",
+                    "branch": "master",
+                    "commit": "f2ccefec47d948191b6060e9b89f5e82bf1307fc"
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -R /run/build/RabbitCommon /var/tmp/"
+            ]
+        },
+        {
+            "name": "RabbitRemoteControl",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/KangLin/RabbitRemoteControl.git",
+                    "branch": "master",
+                    "commit": "930ba87e9b2595839b0bfc168c98cce6979e0bb4"
+                }
+            ],
+            "buildsystem": "cmake-ninja",
+            "builddir": false,
+            "config-opts": [
+                "-DRabbitCommon_ROOT=/var/tmp/RabbitCommon",
+                "-DCMAKE_BUILD_TYPE:STRING=Release",
+                "-DCMARK_SHARED=OFF",
+                "-DCMARK_TESTS=OFF",
+                "-DCMARK_STATIC=ON"
+            ],
+            "post-install": [
+                "sed -i 's|Path=/app/bin|Path=/bin|' /app/share/applications/io.github.KangLin.RabbitRemoteControl.desktop"
```

Comment:
what is this path in desktop file?

Reply:
as I said use the `destkop-file-edit` command instead of sed. But you resolved the comment without actually resolving it. It wasn't a suggestion.